.pre-commit-config.yaml

```diff
@@ -1,11 +1,16 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     sha: master
     hooks:
-    -   id: check-added-large-files
-        args: ['--maxkb=100']
-    -   id: check-case-conflict
-    -   id: end-of-file-fixer
-    -   id: fix-encoding-pragma
-    -   id: mixed-line-ending
-        args: [--fix=lf]
+      - id: check-added-large-files
+        args: [ '--maxkb=100' ]
+      - id: check-case-conflict
+      - id: end-of-file-fixer
+      - id: fix-encoding-pragma
+      - id: mixed-line-ending
+        args: [ --fix=lf ]
+  - repo: https://github.com/pycqa/isort
+    rev: 5.6.4
```

Comment:
any reason we don't use the current version (5.9.3?)

Reply:
no, I just copied what they had in their documentation.

Reply:
pre-commit autoupdate will fix that right up

should we also add the `trailing-whitespace` hook from the main pre-commit default repo? that will get rid of quite a bit of our style errs.

Reply:
I'm good w/ adding that; but I'm not convinced it should happen in this PR given the size of it (also our code quality issues look like they're going to go up by ~70 :( )